### ISSUE TITLE
Coming to Desktop? 

### ISSUE THREAD

**Issue by theotherguy2175** [jamie-mh/AuthenticatorPro#484]
**Describe the solution you'd like**
I love this better than Authy but it would be nice to have either a chrome extension or a desktop app for windows too.

**Comment by xerta555** [comment]
Being a new user of this application since the recent Google Authenticator updates, i also got this idea when i saw in my mind the idea of syncing with cloud solutions like the Stylish extension for example.

The idea is to be able to use different cloud services like Drive, Dropbox and others as backup/synchronization between multiple devices.

**Comment by MagicLike** [comment]
> The idea is to be able to use different cloud services like Drive, Dropbox and others as backup/synchronization between multiple devices.

See the [FAQ](https://github.com/jamie-mh/AuthenticatorPro/wiki#frequently-asked-questions):
> ➡️ Can you add sync to Google Drive, OneDrive, etc...

No. Syncing to cloud services would require the internet permission and heavy intrusive SDKs to support them. Also, they have a habit of changing the APIs often, which just requires more work.

You would have to sync manually or via auto-backup over the Cloud (e.g. Nextcloud) and then auto-restore on the desktop app / browser extension.

**Comment by theotherguy2175** [comment]
No worries I just switched to bitwarden now that it offers otp syncing

**Comment by MagicLike** [comment]
Yeah, I am currently using KeepassXC as my Desktop OTP provider, but I think for example for edits it would be more comfortable to also have a desktop version of AP...

**Comment by MarkToon** [comment]
Hi, just wondering if there has been any advanced talks with regard to a windows based desktop application?

Especially with Authy now looking to close their desktop apps, this is prime time for one of the top open source 2FA apps to gain some big ground.

Here's hoping

**Comment by theotherguy2175** [comment]
i just use bitwarden. and i use this app as a backup for bitwarden

**Comment by avipars** [comment]
> i just use bitwarden. and i use this app as a backup for bitwarden
> […](#)

Requires bitwarden premium?

**Comment by banlanzs** [comment]
I used Claude Code to build a Windows application based on the original project. The interface isn't as aesthetically pleasing as the mobile version, but it's basically functional.

<img width="1257" height="891" alt="Image" src="https://github.com/user-attachments/assets/4501e0d8-3292-4ac7-80b9-fa6c66ddfe20" />
